### DISCUSSION TITLE
ts vom DP nutzen für JSON-Format / Binding

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#1972] [Q&A]
@Zefau 
Kann man auch den Wert von `ts` vom DP nutzen? 
Es geht darum, dass man dann feststellen könnte, ob ein DP über längere Zeit an oder aus ist und dann z.B. einen Effekt auslösen kann.

Zurzeit übergebe ich den Wert:
`{"{0_userdata.0.DP.condition}=true":"motion-sensor alert"}`
`{"true":"motion-sensor alert"}`

`{"now()-{ts}>60000":"motion-sensor alert"}` ?
Kann man da schon den ts-Wert abgreifen?
Und dann `now()-ts >= 60000` abfragen.

**Comment by Zefau** [comment]
Aktuell nicht, aber eine interessante Idee. Mach gerne einen Feature Request auf.

**Reply by Zefau** [reply]
https://github.com/Zefau/ioBroker.jarvis/issues/1973
